added non-well foundedness note

franconi opened the pull request:
Added a technical note discussing why non-well founded interpretations are harmless in RDF


<!--
    This comment and the below content is programmatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-semantics/pull/168.html" title="Last updated on Dec 18, 2025, 9:15 AM UTC (1f4793e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-semantics/168/294ddeb...1f4793e.html" title="Last updated on Dec 18, 2025, 9:15 AM UTC (1f4793e)">Diff</a>

franconi committed:
added non-well foundedness note

franconi committed:
Correct <br> tag in index.html

Fix incorrect closing tag for <br> element.

franconi committed:
Fix HTML encoding for triple example

franconi committed:
Fix HTML escaping in example triple

franconi committed:
Refactor explanation of non-well founded models

Removed redundant text and improved clarity in the explanation of non-well founded models.

franconi committed:
Refactor self-referential paradox explanation

Replaced an ordered list with inline text for clarity.

franconi committed:
Fix typo in index.html from 'paradoxe' to 'paradox'

franconi committed:
Update explanation of triple term mapping in index.html

Clarify the mapping of the triple term in the interpretation.

franconi committed:
Refine explanation of triple term mapping interpretation

Clarified the interpretation of the triple term mapping in the context of RDFS semantic conditions.

franconi committed:
Refactor RDFS interpretations section in index.html

Removed duplicate sentence about RDFS interpretations and added clarification about IRIs.

pfps commented on spec/index.html:
```diff
@@ -1515,6 +1515,31 @@ <h3>RDFS entailment</h3>
 
+	<p class="technote"> 
+		The RDFS semantic condition on the triple term mapping IT induces 
+		unwanted <i>non-well founded models</i> for propositions.		
```
```suggestion
		potentially unwanted <i>non-well founded models</i> for propositions.		
```

TallTed reviewed (dismissed):
Here, again, we have formatting (most line breaks) to make the *source* clearer to reviewers. Similar formatting should be applied to the content being rendered to readers of the ReSpec *output*, to similarly increase comprehension. I have included such in my change requests on this PR. Use your favorite DIFF tool to reveal what GitHub does not.

franconi committed:
weakening the statement

Co-authored-by: Peter F. Patel-Schneider <pfpschneider@gmail.com>

franconi committed:
from <p> to <div>

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

franconi committed:
formatting

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

franconi committed:
Remove empty div from index.html

franconi committed:
Improve clarity in semantic condition explanation - with an added comma

Clarified the explanation of the interpretation of the triple term mapping.

franconi committed:
added another comma for clarity

pchampin reviewed (changes requested):
I have some issues with the very beginning of this note (and with its location), but the rest of the note looks good.

franconi commented:
Ok ok, I agree.
But still I would keep the notion of proposition in this text.

doerthe commented:
I missed the  whole discussion here. I am not sure that we need  the remark at all (but I acn be convinced). Yes we have some possible self-reference in what you call "intended meaning", but I somehow do not see why we want to mention it.

franconi commented:
> I missed the whole discussion here. I am not sure that we need the remark at all (but I can be convinced). Yes we have some possible self-reference in what you call "intended meaning", but I somehow do not see why we want to mention it.

We have to mention this fact in RDF-Semantics since the current _normative_ semantics does not fully capture the intended meaning of a triple term as a _proposition_. We also observe that the unintended models induced by the normative semantics have no impact on the behaviour of simple, RDF, and RDFS entailments.
It is still debatable where in the document this would better fit.

franconi committed:
Moved non-well-foundedness discussion to a nens appendix on semantic adequacy

The previous section discussing non-well founded models has been removed, and a new section on semantic adequacy considerations has been added, explaining the implications of the triple term mapping IT.

franconi committed:
fixed newline

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

franconi committed:
Fixed newline

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

franconi committed:
Added comma

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

franconi committed:
Fixed wording

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

franconi commented:
@pchampin, is it better now?

pchampin reviewed (approved):
Looks good now, thanks.
